### ISSUE TITLE
Crashes when Discord isn't open

### ISSUE THREAD

**Issue by hotsno** [Arno500/plex-richpresence#11]
When the program is running while Discord isn't open, and then I start playing something on Plex, the program crashes. Here's the output I get in the terminal:
```
goroutine 56 [running]:
log.Panicln({0x14000798958?, 0x3?, 0x10?})
	/opt/homebrew/Cellar/go/1.18.2/libexec/src/log/log.go:399 +0x68
gitlab.com/Arno500/plex-richpresence/discord.InitDiscordClient()
	/Users/hotsno/Developer/plex-richpresence/discord/main.go:26 +0x64
gitlab.com/Arno500/plex-richpresence/discord.SetRichPresence({{{0x14000388b60, 0x3}, {0x14000388b66, 0x7}, 0x15b28d, {0x0, 0x0, 0x0}, 0x7, 0x1, ...}, ...}, ...)
	/Users/hotsno/Developer/plex-richpresence/discord/main.go:149 +0xc88
gitlab.com/Arno500/plex-richpresence/plex.StartWebsocketConnections.func2({{0x0, 0x0, 0x0}, {0x0, 0x0, 0x0}, {0x0, 0x0, 0x0}, {0x140002fc780, ...}, ...})
	/Users/hotsno/Developer/plex-richpresence/plex/main.go:220 +0x39c
github.com/Arno500/go-plex-client.(*Plex).SubscribeToNotifications.func1()
	/Users/hotsno/go/pkg/mod/github.com/!arno500/go-plex-client@v1.1.7/websocket_client.go:234 +0x14c
created by github.com/Arno500/go-plex-client.(*Plex).SubscribeToNotifications
	/Users/hotsno/go/pkg/mod/github.com/!arno500/go-plex-client@v1.1.7/websocket_client.go:205 +0x228
```

**Comment by Arno500** [comment]
Hi @hotsno,
Nice point, I didn't even check how it'd work (or not) when Discord was closed. I'll add that to the backlog of when I have time :)
Thanks for the report!

**Comment by Arno500** [comment]
Should be fixed in 1.6.0, now!
